org.adempiere.base/src/org/compiere/model/X_AD_Tab.java

```diff
@@ -307,18 +321,28 @@ public int getAD_Table_ID ()
 		return ii.intValue();
 	}
 
-	/** Set AD_Tab_UU.
-		@param AD_Tab_UU AD_Tab_UU	  */
-	public void setAD_Tab_UU (String AD_Tab_UU)
+	/** AD_TabType AD_Reference_ID=200117 */
+	public static final int AD_TABTYPE_AD_Reference_ID=200117;
+	/** Data Grid = FORM */
+	public static final String AD_TABTYPE_DataGrid = "FORM";
+	/** Order Tab = SORT */
+	public static final String AD_TABTYPE_OrderTab = "SORT";
```

Comment:
looks kind of strange to me that the list name and constant name is completely different. I means it looks a lot more natural if it is AD_TABTYPE_Form="FORM" and AD_TABTYPE_SortTab="SORT"

Reply:
@hengsin originally the isSortTab column is named as Order Tab, So we followed same conventions. Do you think it is good to fix and change label to to Sort?

Reply:
@hengsin this fix committed in comitt https://github.com/idempiere/idempiere/pull/579/commits/7eb84b31564af975ef7a6aad48fcbbf5e1b07c6a
Also resolved merge conflict with Master branch.